packages/app/src/data-sync/reducers/data-sync.reducer.ts

```diff
@@ -41,12 +43,14 @@ export const dataSyncReducer = createReducer<DataSyncState>(
         return {
           ...state,
           state: SynchronizationState.Loading,
+          synchronizationProcess: SynchronizationProcessState.InProgress,
```

Comment:
I wonder if this additional field is needed, cause SynchronizationProcessState.InProgress is only when SynchronizationState.Loading, in other cases synchronizationProcess is SynchronizationProcessState.Done? Can't we calculate synchronizationProcess out of state?

Reply:
I wanted to use `state`. Unfortunately, there is a path that does not fit this case :/